CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+## Setup for Code Contributions
+
+### Prerequisites
+
+##### MacOS
+- XCode Command Line Tools `xcode-select --install`
+- If you already have the full Xcode installed, go to menu `Xcode -> Open Developer Tool -> More Developer Tools....`
+- Install Node.js >= 16
+
+##### Windows
+- Install current version of Python3
+- Install Visual Studio Code and add the Microsoft C/C++ extension
+- Install Node.js >= 16
+
+##### Linux
+
+- Install Python > 3.7
+- Install `make`, `gcc`, and `g++`
+- Install Node.js >= 16
+
+#### Repository Setup
+- Fork the project.
+- Open a Terminal and navigate to where you want the project repository to exist
+- Clone your forked repository
+    ```sh
+    git clone https://github.com/<your-github-user>/club_website.git
```

Comment:
They won't be in the git repo after this step is done, so the others will result in errors. Maybe change it to `git clone https://github.com/<your-github-user>/club_website.git && cd club_website`, or add a separate step for the cd.

Reply:
addressed below

Reply:
Okay, yeah, I'm doing my best here, but I don't have the full picture of how to set this up as there was hiccups even when I did my first setup with creating the beta branch. That might be the final issue to resolve for this.